<launch>
	<node name="mapper" type="mapper" pkg="ethzasl_icp_mapper" output="screen">
		<remap from="cloud_in" to="/rotating_lidar_to_pointcloud_transformer/assembled_laser_pointcloud" />
		<param name="subscribe_scan" value="false" />
		<param name="icpConfig" value="$(find ethzasl_icp_mapper)/launch/multiSenseSL/icp.yaml" />
		<param name="inputFiltersConfig" value="$(find ethzasl_icp_mapper)/launch/multiSenseSL/input_filters.yaml" />
		<param name="mapPreFiltersConfig" value="$(find ethzasl_icp_mapper)/launch/multiSenseSL/map_pre_filters.yaml" />
		<param name="mapPostFiltersConfig" value="$(find ethzasl_icp_mapper)/launch/multiSenseSL/map_post_filters.yaml" />
		<param name="odom_frame" value="/world" />
		<param name="map_frame" value="/world_icp" />
		<param name="useROSLogger" value="true" />
		<param name="minOverlap" value="0.6" /> 
		<param name="maxOverlapToMerge" value="0.9" /> 
		<param name="minReadingPointCount" value="1000" /> 
		<param name="minMapPointCount" value="1000" /> 
		<param name="localizing" value="true" /> 
		<param name="mapping" value="true" />
		<param name="tfRefreshPeriod" value="0.01" />
		<param name="inputQueueSize" value="0" />
	</node>
</launch>
